README.md

```diff
@@ -34,14 +37,50 @@ You *do not* need to build to deploy the code, that is automatic.  See more info
    To ensure that you are open a TypeScript file in VSC and then click on the version number next to
    `TypeScript React` in the status bar and select 'Use Workspace Version' in the popup menu.
 
-## Deployment
+## Deployment to Concord Consortium AWS S3 bucket (https://soundwaves.concord.org)
 
 Production releases to S3 are based on the contents of the /dist folder and are built automatically by GitHub Actions
 for each branch pushed to GitHub and each merge into production.
 
-Merges into production are deployed to http://soundwaves.concord.org.
+Merges into production are deployed to https://soundwaves.concord.org.
+
+Other branches are deployed to `https://soundwaves.concord.org/branch/<name>/index.html`, for example: https://soundwaves.concord.org/branch/master/index.html.
```

Comment:
```suggestion
Other branches are deployed to `https://soundwaves.concord.org/branch/<name>/`, for example: https://soundwaves.concord.org/branch/master/ (note the trailing slash).
```

Reply:
I didn't know someone made that work, nice. 👍 
Interesting, when I try `https://soundwaves.concord.org/branch/master`, it redirects to `https://soundwaves.concord.org/soundwaves/branch/master/` (which is ofc 404).

Reply:
> I didn't know someone made that work, nice. 👍 Interesting, when I try `https://soundwaves.concord.org/branch/master`, it redirects to `https://soundwaves.concord.org/soundwaves/branch/master/` (which is ofc 404).

That trailing slash requirement has been an annoyance for quite some time now but no one's ever gotten around to doing anything about it. 😉